test/unit/DependencyInjection/OpenApiServerExtensionTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\DependencyInjection;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use OnMoon\OpenApiServerBundle\DependencyInjection\OpenApiServerExtension;
+use OnMoon\OpenApiServerBundle\Specification\SpecificationLoader;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Throwable;
+
+/**
+ * @covers \OnMoon\OpenApiServerBundle\DependencyInjection\OpenApiServerExtension
+ */
+class OpenApiServerExtensionTest extends AbstractExtensionTestCase
+{
+    /**
+     * @return array<ExtensionInterface>
+     */
+    protected function getContainerExtensions(): array
+    {
+        return [
+            new OpenApiServerExtension(),
+        ];
+    }
+
+    /**
+     * @return array<int, array<int, string|bool>>
```

Comment:
Psalm has a special type `list` for numeric arrays that have no gaps in keys, it can be used for better readability everywhere you use array<int, ...>
```php
/**
 * @return list<list<string|bool>>
 */
```